packages/svelte/scripts/process-messages/index.js

```diff
@@ -404,3 +404,4 @@ transform('client-errors', 'src/internal/client/errors.js');
 transform('server-errors', 'src/internal/server/errors.js');
 transform('shared-errors', 'src/internal/shared/errors.js');
 transform('shared-warnings', 'src/internal/shared/warnings.js');
+transform('legacy-warnings', 'src/legacy/warnings.js');
```

Comment:
we don't need a new file for this, it should go in `compile-errors/script.md`

Reply:
oh wait my bad this is a runtime error. i meant `client-warnings/warnings.md`

Reply:
It's not a compiler error though and that file gets generated in `src/compiler` not `src/legacy`?

Reply:
Gotcha